public/app/types/events.ts

```diff
@@ -85,7 +85,6 @@ export interface PanelChangeViewPayload {}
 
 export const dsRequestResponse = eventFactory<DataSourceResponsePayload>('ds-request-response');
 export const dsRequestError = eventFactory<any>('ds-request-error');
-export const toggleSidemenuMobile = eventFactory('toggle-sidemenu-mobile');
```

Comment:
nit: Not sure if this is used externally by any plugins.

Reply:
@hugohaggmark hmmmmm... good point. the problem is it won't do anything anymore after this PR 🤔 

we could reimplement the event listener in `NavBar`/`NavBarNext` and use it to toggle the state variable... but that feels a bit icky. 🤮 

have we ever documented this event as part of our public api?

Reply:
I'm just being overly cautious, it's probably safe to remove.